libnext/src/parsers/nodes/api-parser.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { Project, SourceFile } from "ts-simple-ast";
+import { parseApi } from "./api-parser";
+
+describe("@api parser", () => {
+  const apiClassName = "MyApi";
+  const apiName = "My API";
+
+  test("parses all information", () => {
+    const apiDescription = "Some description";
+    const content = `
+      /** ${apiDescription} */
```

Comment:
I'd feel a bit more confident in these tests if we didn't use string interpolation (just hardcoding the constants, duplicating them here). I know that's less code reuse, but these are tests and we should try to minimise unnecessary logic as much as we can :)